tendrl/commons/utils/central_store/utils.py

```diff
@@ -2,7 +2,7 @@
 import datetime
 import etcd
 import sys
-import thread
+import _thread
```

Comment:
```suggestion
import _thread as thread
```
then you don't have to change anywhere else

Reply:
Sure thank you